prdoc/1.9.0/pr_3513.prdoc

```diff
@@ -5,6 +5,12 @@ title: Fix call enum's metadata regression
 
 doc:
   - audience: Runtime Dev
+    description: |
+      This PR fixes an issue where in the metadata of all FRAME-based chains, the documentation for
+      all extrinsic/call/transactions has been replaced by a single line saying "see Pallet::name".
+
+      Many wallets display the metadata of transactions to the user before signing, and this bug
+      might have affected this process.
   - audience: Runtime User
```

Comment:
```suggestion
  - audience: [Runtime User, Runtime Dev]
```

Added this syntax recently, so you dont have to copy&paste it 😄 (and made the description required)